llmfoundry/cli/cli.py

```diff
@@ -1,19 +1,21 @@
 # Copyright 2024 MosaicML LLM Foundry authors
 # SPDX-License-Identifier: Apache-2.0
 
+from argparse import Namespace
 from typing import Optional
 
 import typer
 
 from llmfoundry.cli import registry_cli
+from llmfoundry.data_prep import convert_dataset_hf_from_args
 from llmfoundry.train import train_from_yaml
 
 app = typer.Typer(pretty_exceptions_show_locals=False)
 app.add_typer(registry_cli.app, name='registry')
 
 
 @app.command(name='train')
-def train(
+def train_from_yaml_cli(
```

Comment:
why do we need to make this name change?

Reply:
not needed